components/containers/core/browser/containers_settings_handler.h

```diff
@@ -46,6 +47,10 @@ class ContainersSettingsHandler : public mojom::ContainersSettingsHandler {
   void RemoveContainer(const std::string& id,
                        RemoveContainerCallback callback) override;
 
+  // Returns an error if the given container properties are invalid.
+  static std::optional<mojom::ContainerOperationError>
+  ValidateEditableContainerProperties(const mojom::ContainerPtr& container);
+
   // Returns true if the container name is valid.
   static bool IsContainerNameValid(std::string_view name);
```

Comment:
as a reasonable cleanup I think these separate validators can be removed now and we should only use and test `ValidateEditableContainerProperties`.